ibis/pandas/execution/window.py

```diff
@@ -142,12 +153,21 @@ def execute_window_op(
     pre_executed_scope = pre_execute(
         operand_op, *clients, scope=scope, aggcontext=aggcontext, **kwargs
     )
+    computed_localcontexts = compute_local_context(
+        operand_op, *clients, localcontext=localcontext
+    )
     scope = toolz.merge(scope, pre_executed_scope)
+
+    if len(computed_localcontexts):
```

Comment:
This logic looks really weird, can you explain what is going on?

Reply:
I think I missed this comment. This is explained in another thread since in `window.py` it tries to handle the whole execution of the window manually, and `pre_execute`, `execute` are all called here. We need to add `compute_time_context` here to pass time context to `pre_execute` and `execute`.  Also, this `if` statement is replaced, see latest changes.